pages/learn.js

```diff
@@ -12,6 +12,8 @@ import salishsea from '../images/salishsea.png'
 import audio from '../public/frequency.mp3'
 import orcasound from '../public/frequency.mp3'
 import learnStyles from '../styles/Learn.module.css'
+
+
```

Comment:
Unnecessary whitespace changes (this file shouldn't be in the PR since there are no other edits to it)

Reply:
Reverted